LICENSE

```diff
@@ -0,0 +1,438 @@
+Attribution-NonCommercial-ShareAlike 4.0 International
```

Comment:
I don't think we want the NC variant -- the existing license information in the footer is by-sa, not by-nc-sa.